extension/package.json

```diff
@@ -391,6 +391,12 @@
                         "Sets the Target State to 'needs-review-l10n'"
                     ]
                 },
+                "NAB.useDictionaryInDTSImport": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Toggles usage of substitution dictionary when importing translation files from DTS.",
+                    "scope": "resource"
+                },
```

Comment:
I think default `true` is justifiable but it should probably be combined with `TranslationMode.dts` since that is the only mode we're supporting. What do you think @jwikman?

Reply:
Combined in `LanguageFunctionsSettings.useDTSDictionary`